src/ipc/capnp/chain.capnp

```diff
@@ -0,0 +1,196 @@
+    havePruned @30 (context :Proxy.Context) -> (result :Bool);
+    getPruneHeight @31 (context :Proxy.Context) -> (result: Int32, hasResult: Bool);
+    isReadyToBroadcast @32 (context :Proxy.Context) -> (result :Bool);
+    isInitialBlockDownload @33 (context :Proxy.Context) -> (result :Bool);
+    shutdownRequested @34 (context :Proxy.Context) -> (result :Bool);
+    initMessage @35 (context :Proxy.Context, message :Text) -> ();
+    initWarning @36 (context :Proxy.Context, message :Common.BilingualStr) -> ();
+    initError @37 (context :Proxy.Context, message :Common.BilingualStr) -> ();
+    showProgress @38 (context :Proxy.Context, title :Text, progress :Int32, resumePossible :Bool) -> ();
+    handleNotifications @39 (context :Proxy.Context, notifications :ChainNotifications) -> (result :Handler.Handler);
+    waitForNotificationsIfTipChanged @40 (context :Proxy.Context, oldTip :Data) -> ();
+    handleRpc @41 (context :Proxy.Context, command :RPCCommand) -> (result :Handler.Handler);
+    rpcEnableDeprecated @42 (context :Proxy.Context, method :Text) -> (result :Bool);
+    getSetting @43 (context :Proxy.Context, name :Text) -> (result :Text);
+    getSettingsList @44 (context :Proxy.Context, name :Text) -> (result :List(Text));
+    getRwSetting @45 (context :Proxy.Context, name :Text) -> (result :Text);
+    updateRwSetting @46 (context :Proxy.Context, name :Text, update: SettingsUpdateCallback) -> (result :Bool);
+    overwriteRwSetting @47 (context :Proxy.Context, name :Text, value :Text, action :Int32) -> (result :Bool);
+    deleteRwSettings @48 (context :Proxy.Context, name :Text, action: Int32) -> (result :Bool);
+    requestMempoolTransactions @49 (context :Proxy.Context, notifications :ChainNotifications) -> ();
+    hasAssumedValidChain @50 (context :Proxy.Context) -> (result :Bool);
+}
+
+interface ChainNotifications $Proxy.wrap("interfaces::Chain::Notifications") {
+    destroy @0 (context :Proxy.Context) -> ();
+    transactionAddedToMempool @1 (context :Proxy.Context, tx :Data) -> ();
+    transactionRemovedFromMempool @2 (context :Proxy.Context, tx :Data, reason :Int32) -> ();
+    blockConnected @3 (context :Proxy.Context, role: UInt32, block :BlockInfo) -> ();
+    blockDisconnected @4 (context :Proxy.Context, block :BlockInfo) -> ();
+    updatedBlockTip @5 (context :Proxy.Context) -> ();
+    chainStateFlushed @6 (context :Proxy.Context, role: UInt32, locator :Data) -> ();
+}
+
+interface ChainClient $Proxy.wrap("interfaces::ChainClient") {
+    destroy @0 (context :Proxy.Context) -> ();
+    registerRpcs @1 (context :Proxy.Context) -> ();
+    verify @2 (context :Proxy.Context) -> (result :Bool);
+    load @3 (context :Proxy.Context) -> (result :Bool);
+    start @4 (context :Proxy.Context, scheduler :Void) -> ();
+    stop @5 (context :Proxy.Context) -> ();
+    setMockTime @6 (context :Proxy.Context, time :Int64) -> ();
+    schedulerMockForward @7 (context :Proxy.Context, time :Int64) -> ();
```

Comment:
The original param name is `delta_seconds` (where it doesn't make sense, since it's typed), but here it would, since it's not obvious what `Int64` should contain otherwise.

https://github.com/bitcoin/bitcoin/blob/5fe753b56f450b054c42227c5df8346c72447490/src/interfaces/chain.h#L426